django/db/backends/base/base.py

```diff
@@ -36,6 +37,8 @@ class BaseDatabaseWrapper(object):
     ops = None
     vendor = 'unknown'
     SchemaEditorClass = None
+    # Classes instantiated inside __init__().
+    validation_class = BaseDatabaseValidation
```

Comment:
The dummy backend is only customizing `DatabaseFeatures`, the other classes are from `base`. Does it change your view?